package.json

```diff
@@ -9,11 +9,10 @@
     "lint": "tslint -c tslint.json 'packages/teleport-*/src/**/*.ts{,x}'",
     "clean": "lerna run clean",
     "build": "lerna run build",
-    "dev": "node scripts/watcher.js",
-    "dev:core": "lerna run --parallel build:watch:core",
-    "dev:component": "lerna run --parallel build:watch:component",
-    "dev:project": "lerna run --parallel build:watch:project",
-    "dev:packer": "lerna run --parallel build:watch:packer",
+    "build:cjs": "lerna run build:cjs",
+    "build:esm": "lerna run build:esm",
+    "dev": "npm run build:cjs && npm run watch",
```

Comment:
Hey @alexnm can we change these two lines with this
~~~
     "build": "lerna run build --parallel",
    "build:cjs": "lerna run build:cjs --parallel",
    "build:esm": "lerna run build:esm --parallel",
~~~

Reply:
This is decreasing the build time

Reply:
sure, can you do a PR on this?

Reply:
Ha sure, I will update